packages/server/graphql/mutations/createGitHubTaskIntegration.ts

```diff
@@ -68,44 +71,62 @@ export default {
       })
     }
 
-    // RESOLUTION
     const {content: rawContentStr, meetingId} = task
-    const viewerAuth = await dataLoader.get('githubAuth').load({teamId, userId: viewerId})
-
-    if (!viewerAuth) {
+    const [viewerAuth, assigneeAuth, team, teamMembers] = await Promise.all([
+      dataLoader.get('githubAuth').load({teamId, userId: viewerId}),
+      userId && dataLoader.get('githubAuth').load({teamId, userId}),
+      dataLoader.get('teams').load(teamId),
+      dataLoader.get('teamMembersByTeamId').load(teamId)
+    ])
+    const auth = viewerAuth ?? assigneeAuth
+    if (!auth) {
```

Comment:
we have a tight coupling between userId & assigneeAuth & we should tell typescript about that so we don't have to do a nonnullish coercion down below on L122.

```ts
const accessUserId = viewerAuth ? viewerId : assigneeAuth ? userId : null
if (!accessUserId) {
```

then `accessUserId` is guaranteed non-null, which is what we'd expect